examples/add-dir-to-collection.py

```diff
@@ -26,15 +26,15 @@
 dirlist = ['/tmp/test-adddir' ,'/tmp/test-adddir/subdir-1', \
            '/tmp/test-adddir/subdir-2', '/tmp/test-adddir/subdir-3']
 for directory in dirlist:
-    for i in range(3):
+    for i in range(1):
```

Comment:
We either want it to run three times or if we want it to run only once we can just remove the for. I put three here because it adds a few more files and I thought it'd be nice for testing. Either works for me, but if you decide to run only once let's remove the `for` line then

Reply:
reverted back to 3! 1 was a regression